test/controllers/countries_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class CountriesControllerTest < ActionController::TestCase
+  setup do
+    @country = countries(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:countries)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create country" do
+    assert_difference('Country.count') do
+      post :create, country: { battleground: @country.battleground, name: @country.name, stability: @country.stability }
+    end
+
+    assert_redirected_to country_path(assigns(:country))
+  end
+
+  test "should show country" do
+    get :show, id: @country
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @country
+    assert_response :success
+  end
+
+  test "should update country" do
+    patch :update, id: @country, country: { battleground: @country.battleground, name: @country.name, stability: @country.stability }
+    assert_redirected_to country_path(assigns(:country))
+  end
+
+  test "should destroy country" do
+    assert_difference('Country.count', -1) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.